src/IdRegistry.sol

```diff
@@ -318,4 +317,26 @@ contract IdRegistry is TrustedCaller, Signatures, Pausable, EIP712, Nonces {
             sig
         );
     }
+
+    /*//////////////////////////////////////////////////////////////
+                             EXTERNAL VIEW HELPERS
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice Verify a signature from a fid owner.
```

Comment:
nit: Verify that a signature was produced by the custody address that owns an  fid.

Reply:
Changed